README.md

```diff
@@ -16,109 +15,140 @@ The AdoptOpenJDK Build Farm is three things:
+
+![Extra Projects](images/Adopt_Build_Farm_Extra_Repo_Relationships.png)
+
+* [Technical Steering Committee (TSC)](https://github.com/AdoptOpenJDK/TSC) - The Technical Steering Committee and Knowledge Base starting point.
+  * [TSC-confidential](https://github.com/AdoptOpenJDK/TSC-confidential) - To hold any confidential information for the TSC, e.g. Sponsorship Agreements.
+  * [security](https://github.com/AdoptOpenJDK/security) - A private repo for the security team.
+    * [moderation](https://github.com/AdoptOpenJDK/moderation) - A private repo for moderation requests.
+    * [openjdk-security](https://github.com/AdoptOpenJDK/openjdk-security) - A private repo for (adopt)openjdk vulnerability investigations.
+    * [secrets](https://github.com/AdoptOpenJDK/secrets) - A private repo for secrets.
+  * [openjdk-support](https://github.com/AdoptOpenJDK/openjdk-support) - For end-user problems reported with our binary distributions.
+    * [buzz](https://github.com/AdoptOpenJDK/buzz) - Community Outreach and Sponsors.
+    * [blog](https://github.com/AdoptOpenJDK/blog) - Source for our [public blog](https://blog.adoptopenjdk.net).
+
+### AdoptOpenJDK Source Repositories
+
+The following diagram lists the source repositories for AdoptOpenJDK (edit the
+[Image Source](https://docs.google.com/drawings/d/1h2YzP1zAUsYku8sFbx5xP2mTe1DFVx-6NOHzQ3Hso5w/edit) at Google and export to make changes.)
 
-### Clones of OpenJDK Forests
+![Source Repositories](images/Adopt_Build_Farm_Source_Repos.png)
 
 When an OpenJDK variant is mercurial based or AdoptOpenJDK needs to maintain its own patches then we have a clone of that source:
 
 * [openjdk-jdk8u](https://github.com/AdoptOpenJDK/openjdk-jdk8u)
 * [openjdk-aarch64-jdk8u](https://github.com/AdoptOpenJDK/openjdk-aarch64-jdk8u)
 * [openjdk-aarch32-jdk8u](https://github.com/AdoptOpenJDK/openjdk-aarch32-jdk8u)
-* [openjdk-jdk8u-jfr-incubator](https://github.com/AdoptOpenJDK/openjdk-jdk8u-jfr-incubator)
 * [openjdk-jdk9u](https://github.com/AdoptOpenJDK/openjdk-jdk9u)
 * [openjdk-jdk10u](https://github.com/AdoptOpenJDK/openjdk-jdk10u)
 * [openjdk-jdk11u](https://github.com/AdoptOpenJDK/openjdk-jdk11u)
 * [openjdk-jdk12u](https://github.com/AdoptOpenJDK/openjdk-jdk12u)
 * [openjdk-jdk13u](https://github.com/AdoptOpenJDK/openjdk-jdk13u)
+* [openjdk-jdk14u](https://github.com/AdoptOpenJDK/openjdk-jdk14u)
+* [openjdk-jdk15u](https://github.com/AdoptOpenJDK/openjdk-jdk15u)
+* [openjdk-jdk16](https://github.com/AdoptOpenJDK/openjdk-jdk16)
 * [openjdk-jdk](https://github.com/AdoptOpenJDK/openjdk-jdk)
 * [openjdk-jfx](https://github.com/AdoptOpenJDK/openjdk-jfx)
 * [openjdk-amber](https://github.com/AdoptOpenJDK/openjdk-amber)
+* [openjdk-amber-nightly](https://github.com/AdoptOpenJDK/openjdk-amber-nightly)
```

Comment:
Not sure this one is kept up to date - The [JavaFutures](https://ci.adoptopenjdk.net/view/Build%20and%20Test%20Pipeline%20Calendar/job/JavaFutures/) job pulls direct from Skara repos now